docs/4.3/kubernetes_ssh.md

```diff
@@ -142,15 +135,15 @@ However, it will result in Teleport proxy using your personal Kubernetes
 credentials. This is risky: your credentials can expire or get revoked (such as
 when leaving your company).
 
-## Impersonation
-
-!!! note
+!!! tip
 
     If you used [the script from Option
     2](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh)
     above, you can skip this step. The script already configured impersonation
     permissions.
 
+## Impersonation
+
```

Comment:
Will the reader understand that the tip talks about skipping the `Impersonation` section?

Reply:
I double checked it, and I agree I should move it back.
![image](https://user-images.githubusercontent.com/559288/88350613-7aafde80-cd08-11ea-9adc-a972974a4ea3.png)